src/react-disposable-decorator.js

```diff
@@ -9,7 +9,7 @@ if (!enabled) {
 
 export default function(DecoratedComponent) {
 	if (!enabled) {
-		return DecoratedComponent;
+		return <DecoratedComponent cancelWhenUnmounted={() => {}} />;
```

Comment:
yes, joel explained to me what the difference was. Turns out right now there is no good solution to this problem that we can think of. How do you cancel a pr?